ingest_api/infrastructure/config.py

```diff
@@ -36,18 +34,9 @@ class IngestorConfig(BaseSettings):
     client_secret: Optional[str] = Field(
         "", description="The Cognito APP client secret"
     )
-
-    stac_db_secret_name: str = Field(
-        description="Name of secret containing pgSTAC DB connection information"
-    )
-    stac_db_vpc_id: str = Field(description="ID of VPC running pgSTAC DB")
     stac_db_security_group_id: str = Field(
         description="ID of Security Group used by pgSTAC DB"
     )
-    stac_db_public_subnet: bool = Field(
-        description="Boolean indicating whether or not pgSTAC DB is in a public subnet",
-        default=True,
-    )
```

Comment:
these are not being used anymore, but directly passed to the Ingestor Constructs